api/controllers/gamestate/points.js

```diff
@@ -0,0 +1,25 @@
+// api/controller/game/draw.js
+// Request to draw a card
+module.exports = async function (req, res) {
+  try {
+    const game = await sails.helpers.lockGame(req.params.gameId);
+    const currentState = sails.helpers.unpackGameState(game.gameStates[game.gameStates.length - 1]);
+    const requestedMove = sails.helpers.moves.points.parseRequest(req);
+
+    // Validate move -- Throws if move is illegal e.g. not your turn
+    sails.helpers.moves.points.validate(currentState, requestedMove);
+
+    // Move cards around, increment turn, etc
+    const updatedState = sails.helpers.moves.points.execute(currentState, requestedMove);
```

Comment:
As I look more closely, I see that these are the only three lines that are specific to the particular move made, and only because of the prefix on the helper names. We could generalize this into a single endpoint for all moves and DRY up the code (while being more RESTful, too)
```suggestion
    const { parseRequest, validate, execute } = sails.helpers.moves[req.body.moveType];
    const requestedMove = parseRequest(req);

    // Validate move -- Throws if move is illegal e.g. not your turn
    validate(currentState, requestedMove);

    // Move cards around, increment turn, etc
    const updatedState = execute(currentState, requestedMove);
```

This will require two other changes outside this file:
1) we need to apply a policy to this endpoint that verifies that req.body has a moveType and that it's valid (ie one of the values in MoveType)
2) we need to update the route for this endpoint and all consumers to instead be simply POST /api/game/:gameId/move (removing /points etc) and expect that the req body include the move type